internal/services/recoveryservices/backup_protected_vm_resource.go

```diff
@@ -59,22 +59,20 @@ func resourceRecoveryServicesBackupProtectedVM() *pluginsdk.Resource {
 	}
 }
 
-func resourceRecoveryServicesBackupProtectedVMCreateUpdate(d *pluginsdk.ResourceData, meta interface{}) error {
+func resourceRecoveryServicesBackupProtectedVMCreate(d *pluginsdk.ResourceData, meta interface{}) error {
 	client := meta.(*clients.Client).RecoveryServices.ProtectedItemsClient
 	opClient := meta.(*clients.Client).RecoveryServices.ProtectedItemOperationResultsClient
 	subscriptionId := meta.(*clients.Client).Account.SubscriptionId
-	ctx, cancel := timeouts.ForCreateUpdate(meta.(*clients.Client).StopContext, d)
+	ctx, cancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
 	resourceGroup := d.Get("resource_group_name").(string)
 
 	vaultName := d.Get("recovery_vault_name").(string)
 
 	// source_vm_id must be specified at creation time but can be removed during update
-	if d.IsNewResource() {
-		if _, ok := d.GetOk("source_vm_id"); !ok {
-			return fmt.Errorf("`source_vm_id` must be specified when creating")
-		}
+	if _, ok := d.GetOk("source_vm_id"); !ok {
+		return fmt.Errorf("`source_vm_id` must be specified when creating")
```

Comment:
I can't comment on the line, but there's a `d.IsNewResource()` check on line 81 which can be removed as well.